test/service/statedb/statedb_test_load.cpp

```diff
@@ -147,23 +147,23 @@ void runStateDBTestLoad (const Config& config)
         uint64_t r;
 
         for (int k=0; k<4; k++) {
-            r = distrib(gen); 
+            r = distrib(gen);
             keyScalar = (keyScalar << 64) + r;
         }
 
         scalar2key(fr, keyScalar, key);
         //stateDB.setDBDebug(true);
-        value=2;        
-        client.set (oldRoot, key, value, true, newRoot, &setResult);
+        value=2;
+        client.set (oldRoot, key, value, true, newRoot, &setResult, NULL);
```

Comment:
idea, for compatibility perspective, would be better last parameter of set and get was by default NULL?